tests/baselines/reference/complexRecursiveCollections.errors.txt

```diff
@@ -223,6 +226,10 @@ immutable.ts(391,22): error TS2430: Interface 'Set<T>' incorrectly extends inter
       export function Stack<T>(): Stack<T>;
       export function Stack<T>(collection: Iterable<T>): Stack<T>;
       export interface Stack<T> extends Collection.Indexed<T> {
+                       ~~~~~

```

Comment:
I have a hard time assessing if this is a good change or not, could use some help with that 😅

Reply:
This is probably a good change. This used to stack out, using more variance calculations removed the stack out, but I think have been hiding the error with incorrect results.